### PR TITLE
Dynamic tax rate for mini cya

### PR DESCRIPTION
**Pull request by mauro-fanton**

### CODE REVIEW

File: app/viewmodels/checkAnswers/returns/credits/CreditsConvertedPlasticSummary.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.checkAnswers.SummaryViewModel
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object CreditsConvertedPlasticSummary extends SummaryViewModel {
+  override def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] = {
+    answers.get(ConvertedCreditsPage).map {
+      answer =>
+        val value = if (answer) "site.yes" else "site.no"
```
Comment (lmcewanhmrc) [inline app/viewmodels/checkAnswers/returns/credits/CreditsConvertedPlasticSummary.scala:33]: Duplicated from CreditsAnswer.yesNo ?

File: app/viewmodels/checkAnswers/returns/credits/CreditsConvertedWeightSummary.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object CreditsConvertedWeightSummary extends SummaryViewModel{
+  override def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] = {
+    answers.get(ConvertedCreditsWeightPage).map {
+      value =>
+        SummaryListRowViewModel(
+          key = Key("confirmPackagingCredit.converted.weight", classes="govuk-!-width-one-half"),
+          value = ValueViewModel(value.asKg),
```
Comment (lmcewanhmrc) [inline app/viewmodels/checkAnswers/returns/credits/CreditsConvertedWeightSummary.scala:35]: Should use CreditsAnswer here too?

File: app/viewmodels/govuk/SummaryListFluency.scala [hunk 3]
```diff
@@ -106,6 +108,11 @@ trait SummaryListFluency {
     def withCssClass(className: String): Value =
       value copy (classes = s"${value.classes} $className")
 
+    def withVisuallyHiddenText(text: String): Value = {
+      val contentWithHiddenText = HtmlContent(Html(s"""<p>${value.content.asHtml.body}<span class="govuk-visually-hidden">$text</span></p>"""))
+      value copy(content = contentWithHiddenText)
+    }
```
Comment (lmcewanhmrc) [inline app/viewmodels/govuk/SummaryListFluency.scala:114]: What does this do? :)

File: conf/messages.cy [hunk 4]
```diff
@@ -3,6 +3,7 @@ service.name = Treth Deunydd Pacio Plastig
 site.back = Yn ôl
 site.remove = Tynnu
 site.change = Newid
+site.cancel = [NEED WELSH] Cancel
```
Comment (lmcewanhmrc) [inline conf/messages.cy:6]: Funny we don't have this already 😄

**Review by lmcewanhmrc — APPROVED** [review]

File: app/pages/returns/credits/CancelCreditsClaimPage.scala [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+package pages.returns.credits
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object CancelCreditsClaimPage extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "CancelCreditsClaim"
```
[the remark this answers is not in the document]
Comment (mauro-fanton) [reply app/pages/returns/credits/CancelCreditsClaimPage.scala:26]: yes agree, no point having the toString

**Pull request merged**